Formula/kubectx.rb

```diff
@@ -0,0 +1,26 @@
+class Kubectx < Formula
+  desc "Tool that can switch between kubectl contexts easily and create aliases"
+  homepage "https://github.com/ahmetb/kubectx"
+  url "https://github.com/ahmetb/kubectx/archive/v0.3.1.tar.gz"
+  sha256 "4e995f5bec6f41c8d5b6e77f413a58ead077816348e72de26dde3655ec2b7d0b"
+  head "https://github.com/ahmetb/kubectx.git", :branch => "master"
+  bottle :unneeded
+
+  option "with-short-names", "link as \"kctx\" and \"kns\" instead"
```

Comment:
The point of this option is to prevent having them prefixed with kube*, so it doesn't collide with kubectl.